students/example_student/DanCornutt/Readme.md

```diff
@@ -0,0 +1,5 @@
+#Dan's dir for examples#
```

Comment:
oops -- you didn't want to put this in the "example_student" dir.

To fix:

`git rm  students/example_student/DanCornutt/Readme.md`

git commit -s -m "removed accidental file"

git push

This PR should be updated automatically when you push to your gitHub repo.